.drone.yml

```diff
@@ -0,0 +1,105 @@
+---
+kind: pipeline
+type: kubernetes
+name: pr
+
+trigger:
+  event:
+  - pull_request
+
+steps:
+  - name: fetch tags
+    image: docker:git
+    commands:
+      - git fetch --tags
+  - name: wait for docker
+    image: docker
+    commands:
+      - timeout 15s /bin/sh -c 'while [ ! -S /var/run/docker.sock ]; do sleep 1; done'
```

Comment:
I'm sure this is copied from somewhere else, but any idea why it's needed?  Unless I'm missing something this is the docker socket bind mounted from the host, which should be mounted far ahead of the container being run?

Reply:
The docker socket is mounted from the following:

```
services:
  - name: run docker daemon
    image: docker:dind
    privileged: true
    volumes:
      - name: dockersock
        path: /var/run
```

Without this, we'll get failures like the following:

https://drone.gravitational.io/gravitational/robotest/4/1/3